.github/CHANGELOG.md

```diff
@@ -242,6 +242,9 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 * Prevent Hamiltonians that share wires from being multiplied together.
   [(#1273)](https://github.com/PennyLaneAI/pennylane/pull/1273)
 
+* Fixed the custom range sequence feature and added test for it.
+  [(#1332)](https://github.com/PennyLaneAI/pennylane/pull/1332)
```

Comment:
It would be good to provide a few more details here, for example
```suggestion
* Fixed a bug where the custom range sequences could not be passed
  to the `StronglyEntanglingLayers` template.
  [(#1332)](https://github.com/PennyLaneAI/pennylane/pull/1332)
```